packages/auth/src/accessToken/routes.test.ts

```diff
@@ -113,40 +111,67 @@ describe('Access Token Routes', (): void => {
       }
     )
     test('Cannot introspect fake token', async (): Promise<void> => {
+      const requestBody = {
+        access_token: v4(),
+        proof: 'httpsig',
+        resource_server: 'test'
+      }
+      const { signature, sigInput, contentDigest } = await generateSigHeaders(
+        url,
+        method,
+        requestBody
+      )
       const ctx = createContext(
```

Comment:
Maybe you could add a wrapper for `createContext` that takes `url`, `method`, `requestBody`, etc as parameters and does `generateSigHeaders` + `createContext` instead of having to do this full setup in all these tests.

Reply:
I think we can actually just remove it since the test doesn't run the middleware.